index.bs

```diff
@@ -3593,14 +3593,20 @@ associated with or [=scoped=] to, respectively.
     :   <dfn>name</dfn>
     ::  A [=human palatability|human-palatable=] name for the entity. Its function depends on what the {{PublicKeyCredentialEntity}} represents:
 
-          - When inherited by {{PublicKeyCredentialRpEntity}} it is a [=human palatability|human-palatable=] identifier for the [=[RP]=], intended only
+          - \[DEPRECATED] When inherited by {{PublicKeyCredentialRpEntity}} it is a [=human palatability|human-palatable=] identifier for the [=[RP]=], intended only
             for display. For example, "ACME Corporation", "Wonderful Widgets, Inc." or "ОАО Примертех".
 
+            This member is deprecated because many [=clients=] do not display it,
+            but it remains a required dictionary member for backwards compatibility.
+            [=[RPS]=] MAY as a safe default set this equal to the [=RP ID=].
```

Comment:
```suggestion
            [=[RPS]=] MAY, as a safe default, set this equal to the [=RP ID=].
```

Reply:
Yeah, the problem with that is that [we implicitly recommend against empty string](https://github.com/w3c/webauthn/issues/2121#issuecomment-2289741275) by reference to the PRECIS profile, so omitting empty string here is a bit more consistent with that.